gnomad/utils/vcf.py

```diff
@@ -92,3 +340,534 @@ def get_pipe_expr(array_expr: hl.expr.ArrayExpression) -> hl.expr.StringExpressi
+            },
+            f"{prefix}age_hist_het_n_smaller": {
+                "Number": "A",
+                "Description": f"Count of age values falling below lowest histogram bin edge for heterozygous individuals{description_text}",
+            },
+            f"{prefix}age_hist_het_n_larger": {
+                "Number": "A",
+                "Description": f"Count of age values falling above highest histogram bin edge for heterozygous individuals{description_text}",
+            },
+            f"{prefix}age_hist_hom_bin_freq": {
+                "Number": "A",
+                "Description": f"Histogram of ages of homozygous alternate individuals{description_text}; bin edges are: {bin_edges['hom']}; total number of individuals of any genotype bin: {age_hist_data}",
+            },
+            f"{prefix}age_hist_hom_n_smaller": {
+                "Number": "A",
+                "Description": f"Count of age values falling below lowest histogram bin edge for homozygous alternate individuals{description_text}",
+            },
+            f"{prefix}age_hist_hom_n_larger": {
+                "Number": "A",
+                "Description": f"Count of age values falling above highest histogram bin edge for homozygous alternate individuals{description_text}",
+            },
+        }
+        info_dict.update(age_hist_dict)
+
+    if popmax:
+        popmax_dict = {
+            f"{prefix}popmax": {
+                "Number": "A",
+                "Description": f"Population with maximum AF{description_text}",
+            },
+            f"{prefix}AC_popmax": {
+                "Number": "A",
+                "Description": f"Allele count in the population with the maximum AF{description_text}",
+            },
+            f"{prefix}AN_popmax": {
+                "Number": "A",
+                "Description": f"Total number of alleles in the population with the maximum AF{description_text}",
+            },
+            f"{prefix}AF_popmax": {
+                "Number": "A",
+                "Description": f"Maximum allele frequency across populations{description_text}",
+            },
+            f"{prefix}nhomalt_popmax": {
+                "Number": "A",
+                "Description": f"Count of homozygous individuals in the population with the maximum allele frequency{description_text}",
+            },
+        }
+        info_dict.update(popmax_dict)
+
+    else:
+        group_types = sorted(label_groups.keys(), key=lambda x: sort_order.index(x))
+        combos = make_label_combos(label_groups)
+
+        for combo in combos:
+            combo_fields = combo.split("_")
+            group_dict = dict(zip(group_types, combo_fields))
+
+            if not faf:
+                combo_dict = {
+                    f"{prefix}AC_{combo}": {
+                        "Number": "A",
+                        "Description": f"Alternate allele count{make_combo_header_text('for', group_dict, prefix)}",
```

Comment:
Not necessary, but `make_combo_header_text('for', group_dict, prefix)` and `make_combo_header_text('to', group_dict, prefix)` are called several times. Could make them variables and use those here.